portal/templates/layouts/courseEdit.html

```diff
@@ -0,0 +1,70 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+
+
+<header>
+  <a href="{{ url_for('courseEditor.course_manage')}}">Back</a>
+</header>
+
+<section class='edit'><!--Start of section edit-->
+
+<h2>Edit page for {{ course['course_title']}}</h2>
+<!--Form for the edit course page
+which has three inputs of data-->
+
+
+
+
+<form class="editFeature" action="{{ url_for( 'courseEditor.course_edit', id=course['course_num'] ) }}" method="post">
+
+  <p>The courses current information is displayed
+  please fill out all the fields and save changes.</p>
+
+<ul>
+
+
+  <label for="editTitle">Course Title</label><!--Bold in styles-->
+
+<li> {{ course['course_title'] }}
+
+
+  <input type="text" name="editTitle" required>
+
+
+
+
+</li>
+
+<label for="editDesc">Course Description</label>
+
+<li>{{ course['description'] }}
+
+<input type="text" name="editDesc" required>
+
+
+
+</li>
+
+  <label for="editCredit">Course Credits</label>
+
+<li>{{ course['credits'] }}
+
+  <input type="text" name="editCredit" required>
+
+
+
+</li>
+
+
+
+</ul>
```

Comment:
the `<label>`s should be inside of the `<li>`s as a label and its corresponding input should be one item on the list
Right now, the unordered list goes:
- label
- list item
- label
- list item
- label
- list item